tests/foreman/api/test_ansible.py

```diff
@@ -253,3 +255,61 @@ def test_add_and_remove_ansible_role_hostgroup(target_sat):
         target_sat.api.HostGroup(id=hg.id).remove_ansible_role(data={'ansible_role_id': role})
     host_roles = target_sat.api.HostGroup(id=hg.id).list_ansible_roles()
     assert len(host_roles) == 0
+
+
+@pytest.mark.rhel_ver_match('[7]')
```

Comment:
why we are doing it on RHEL7 and why not RHEL8? In my view, if we are going to run the test on a single RHEL client then we should target the latest rhel client

Reply:
Ok, not completely. Setup fails for RHEL9 and FIPS and I don't think it's very related to this PR so I only select non-fips RHELs 6-8.